js/main.js

```diff
@@ -0,0 +1,36 @@
+const div = document.createElement('div');
+const grayBgDiv = document.createElement('div');
+const hamburgerBttn = document.querySelector('.menu-bars');
+
+hamburgerBttn.addEventListener('click', () => {
+  div.id = 'overlay';
+  div.classList.add('overlay');
+  grayBgDiv.id = 'overlay-menu-bg';
+  grayBgDiv.classList.add('overlay');
+  div.innerHTML = '<img src="./media/cancel.png" class="cancelBttn"></img>'
+    + '<ul class="mobile-menu-ul">'
+    + '<li class="mobile-menu-li"><a href="#projects">Porfolio</a></li>'
+    + '<li class="mobile-menu-li"><a href="#about-me">About</a></li>'
```

Comment:
- [ ] Please remember the hamburger button and menu must follow the design from the chosen template in Figma (e.g., font, colors, images, tests, margins). in this case, your project has a little difference in the menu as you can see in the following images:

Your project: 
![Captura de pantalla 2021-12-22 164612](https://user-images.githubusercontent.com/82678907/147159129-4b8c0d1d-9bb5-4152-abf3-7c4d05fa8225.png)

Figma Design:
![Captura de pantalla 2021-12-22 164654](https://user-images.githubusercontent.com/82678907/147159147-8e4532ee-e388-427b-9576-d6f81bd78135.png)

Kindly fix this issue.

Reply:
Done ✔